packages/core/src/lib/implementation/execute-plugin.unit-test.ts

```diff
@@ -1,30 +1,34 @@
 import { vol } from 'memfs';
-import { describe, expect, it } from 'vitest';
+import { describe, expect, it, vi } from 'vitest';
 import { AuditOutputs, PluginConfig } from '@code-pushup/models';
 import {
   MEMFS_VOLUME,
   MINIMAL_PLUGIN_CONFIG_MOCK,
 } from '@code-pushup/test-utils';
+import { ui } from '@code-pushup/utils';
 import {
   PluginOutputMissingAuditError,
   executePlugin,
   executePlugins,
 } from './execute-plugin';
 
+const nodePluginSlug = MINIMAL_PLUGIN_CONFIG_MOCK.slug;
```

Comment:
Repeating a value is okay in tests. If you are worried that one might not know where the value comes from, you can reference the `MINIMAL_PLUGIN_CONFIG_MOCK.slug` in the tests but in general, hardcoded values are okay.

```suggestion
```

Reply:
done. used static values